apps/src/music/views/BeatPad.jsx

```diff
@@ -1,72 +1,74 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import {useSelector} from 'react-redux';
 import classNames from 'classnames';
 import styles from './beatpad.module.scss';
-import FontAwesome from '@cdo/apps/templates/FontAwesome';
-import musicI18n from '../locale';
+import {BlockTypes} from '../blockly/blockTypes';
+import {TRIGGER_FIELD} from '../blockly/constants';
 
-const BUTTONS_PER_ROW = 3;
+const BUTTONS_PER_ROW = 4;
 const enabledClasses = [
-  styles.purple,
-  styles.strawberry,
-  styles.orange,
-  styles.yellow,
-  styles.green,
-  styles.teal,
+  styles.greenTrigger,
+  styles.greenTrigger,
+  styles.greenTrigger,
+  styles.greenTrigger,
+  styles.greenTrigger,
+  styles.greenTrigger,
 ];
 
 /**
  * Renders the Beat Pad component, which can be used to play numbered triggers during playback
  */
-const BeatPad = ({triggers, playTrigger, onClose, isPlaying}) => {
-  const renderTriggers = () => {
-    const rows = [];
+const BeatPad = ({triggers, playTrigger, onClose, isPlaying, hasTrigger}) => {
+  const selectedBlockId = useSelector(state => state.music.selectedBlockId);
+  const selectedBlock =
+    selectedBlockId && Blockly.mainBlockSpace.getBlockById(selectedBlockId);
+  const isSelectedBlockTriggerAt =
+    selectedBlock?.type === BlockTypes.TRIGGERED_AT_SIMPLE2;
+  const selectedTriggerId =
+    isSelectedBlockTriggerAt && selectedBlock?.getFieldValue(TRIGGER_FIELD);
 
-    for (let i = 0; i < triggers.length; i += BUTTONS_PER_ROW) {
-      let buttons = [];
-      for (let j = i; j < i + BUTTONS_PER_ROW; j++) {
+  const renderTriggers = () => {
+    let buttons = [];
+    for (let j = 0; j < BUTTONS_PER_ROW; j++) {
+      if (hasTrigger(triggers[j].id)) {
         buttons.push(
           <TriggerButton
             label={triggers[j].buttonLabel}
             onClick={() => playTrigger(triggers[j].id)}
             key={triggers[j].id}
-            colorClassName={classNames(isPlaying && enabledClasses[j])}
+            colorClassName={classNames(/*isPlaying && */ enabledClasses[j])}
```

Comment:
Is this comment (`/* isPlaying && */`) intentional?

Reply:
Oh, I'll remove the commented-out code for now.  I think there is a more nuanced design discussion to be had around these buttons, but for now I've just got the boldest appearance all the time.